browser/resources/settings/email_aliases_page/content/styles/Row.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import styled from "styled-components"
+
+const Row = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  column-gap: 1em;
```

Comment:
probably not intended by the designer

Reply:
(looks like maybe `spacing.xl` judging by some of the gaps they've used)

Reply:
removed